scripts/build/build.sh

```diff
@@ -24,6 +24,9 @@ header "Checking Pre-Reqs"
 
 $REPOROOT/scripts/test/check-prereqs.sh
 
+# Increases the file descriptors limit for this bash. It prevents an issue we were hitting during restore
+ulimit -n 512
```

Comment:
After thinking about this some more, on some systems we may actually be reducing the max file limit. For example on my Ubuntu vm, the max is 1024. 

Should we do some checks to see which OS we're on?